docs/FAQ.md

```diff
@@ -109,10 +109,41 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I deal with divergent changes ('??' after the ID)?
+
+A [divergent change][glossary_divergent_change] represents a change that has two
+or more visible commits associated with it. To refer to such commits, you must
+use their [commit ID][glossary_commit_id]. Most commonly, the way to resolve
+this is to abandon the unneeded commits (using `jj abandon <commit ID>`). If you
+would like to keep both revisions of the change, you can `jj duplicate` one of
+them before abandoning it.
+
+Usually, the different commits associated with the divergent change ID should all
+appear in the log, but due to #2476, they may not. If that happens, you can
+either use `jj log -r 'all()' | grep <change id>` or disable the
+`revsets.short-prefixes` config option.
+
+### How do I deal with conflicted branches ('?' after branch name)?
+
+A [conflicted branch][branches_conflicts] is a branch that refers to multiple
+different commits that can't be resolved automatically. Resolving conflicted
```

Comment:
(Optional) Instead of "that can't be resolved automatically", we could write "because `jj` couldn't fully resolve its desired position"